tests/authorization/conftest.py

```diff
@@ -54,6 +61,10 @@ def pytest_addoption(parser):
 def build_dir(request):
     return request.config.option.build_dir
 
+@pytest.fixture(scope="session")
+def external_lib_path(build_dir):
+    return os.path.join(build_dir, "bindings/c/libfdb_c_external.so")
```

Comment:
Make sure the test is restricted to Linux only

Reply:
It already skips adding test for `WIN32`. I can add `dylib` case for Mac.